docs/chapters/05-3.md

```diff
@@ -0,0 +1,278 @@
+---
+lang-ref: ch.05-3
+title: Understanding Convolutions and Automatic Gradient
+authors: Leyi Zhu, Siqi Wang, Tao Wang, Anqi Zhang
+date: 28 Feb 2020
+---
+
+## Understanding Matrix-Vector Mulplication
+
+$$
+A \boldsymbol{x}=\left[\begin{array}{cccc}
+{a_{11}} & {a_{11}} & {\cdots} & {a_{1 n}} \\
+{a_{21}} & {a_{22}} & {\cdots} & {a_{2 n}} \\
+{\vdots} & {\vdots} & {\ddots} & {\vdots} \\
+{a_{m 1}} & {a_{m 2}} & {\cdots} & {a_{m n}}
+\end{array}\right]\left[\begin{array}{c}
+{x_{1}} \\
+{x_{2}} \\
+{\vdots} \\
+{x_{n}}
+\end{array}\right]
+$$
+
+There are two different equivalent representations of matrix-vector multiplication. The first one is 
+
+$$
+A\boldsymbol{x}=\left[\begin{array}{c}
+{\boldsymbol{a}^{(1)} \boldsymbol{x}} \\
+{\boldsymbol{a}^{(2)}\boldsymbol{x}} \\
+{\vdots} \\
+{\boldsymbol{a}^{(m)} \boldsymbol{x}}
+\end{array}\right]
+$$
+
+Where $\boldsymbol{a}^{(i)}$ is $i$th row vector of matrix $A$ and each $\boldsymbol{a}^{(i)}\boldsymbol{x}$ can be viewed as the **projection** from  input signal space to the kernel. 
+
+The other interpretation is 
+
+$$
+A\boldsymbol{x}=\left[\begin{array}{c}
+{a_{11}} \\
+{a_{21}} \\
+{\vdots} \\
+{a_{m 1}}
+\end{array}\right] x_{1}+\left[\begin{array}{c}
+{a_{12}} \\
+{a_{22}} \\
+{\vdots} \\
+{a_{m 2}}
+\end{array}\right] x_{2}+\ldots \ldots+\left[\begin{array}{c}
+{a_{1 n}} \\
+{a_{2 n}} \\
+{\vdots} \\
+{a_{m n}}
+\end{array}\right] x_{n}
+$$
+
+This can be viewed as weighted sum of the columns of matrix $A$.
```

Comment:
We covered this in the previous lab.
See https://atcold.github.io/pytorch-Deep-Learning/chapters/04-1/.
Perhaps it's not worth repeating ourselves on the website?